packages/builder/test/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig.json",
+  "include": [
+    "../src",
```

Comment:
Eslint does check both sources. So this config works as long as it isn't used to compile the files.